week4/week4-082.Lottery/src/LotteryNumbers.java

```diff
@@ -0,0 +1,35 @@
+import java.util.ArrayList;
+import java.util.Random;
+
+public class LotteryNumbers {
+    private ArrayList<Integer> numbers;
+    private int number;
+    private Random random;
+
+    public LotteryNumbers() {
+        this.numbers = new ArrayList<Integer>();
+        this.random = new Random();
+        this.drawNumbers();
+    }
+
+    public ArrayList<Integer> numbers() {
+        return this.numbers;
+    }
+
+    public void drawNumbers() {
+        this.numbers = new ArrayList<Integer>();
+        int i = 0;
+        while(i < 7){
+            number = random.nextInt(39)+1;
```

Comment:
`number = random.nextInt(39)+1;`

Ovo 39 se zove "magic number" ili "magic constant". Inače ih je poželjno stavljati u konstante i na taj način im davati neko smisleno ime. Pogotovo ako imaš potrebu da na više mjesta pišeš 39 (ili da u jednom momentu promijeniš). 

U ovako malom kodu to i nije toliki problem, ono, više je da znaš. Desiće se da na 3 mjesta napišeš magični broj i da zatrebaš da promijeniš, promijenićeš ga na 2 i na jednom ćeš zaboraviti i eto bug.